isamples_metadata/Transformer.py

```diff
@@ -21,13 +29,25 @@ def _transform_key_to_label(
         if label is None:
             label = key
         value = source_dict.get(key)
-        if value is not None:
+        if value is not None and len(value) > 0:
```

Comment:
One major difference is that the Smithsonian records (at least in my tests) had empty strings.  I should probably fix this on the frontend things import to not include empty string values.  That would make our lives easier downstream here…